tests/outputs/expected/sliced_housing_simple.py

```diff
@@ -1,9 +1,3 @@
-import pickle
-
-import pandas as pd
-from sklearn.ensemble import RandomForestClassifier
-
-
```

Comment:
These lines got accidentally deleted when merging import fix, which is why the tests are failing again. We need to get them back. Same for other pipeline-related files as well.

Reply:
Seems like test files got modified after the merge.